app/src/main/java/com/android/sample/model/map/LocationViewModel.kt

```diff
@@ -4,6 +4,7 @@ import android.util.Log
 import androidx.lifecycle.ViewModel
 import dagger.hilt.android.lifecycle.HiltViewModel
 import javax.inject.Inject
+import kotlin.math.*
```

Comment:
Importing the **whole** math library is useful here as you use 6 different math functions. If you used less, I'd have recommended to only import the necessary functions.